src/action/info.js

```diff
@@ -14,16 +17,41 @@ class InfoAction {
       this._store.pubKey = response.identity_pubkey;
       this._store.syncedToChain = response.synced_to_chain;
       this._store.blockHeight = response.block_height;
+      if (this.firstTime) {
+        this.startingSyncTimestamp = response.best_header_timestamp || 0;
+        this.firstTime = false;
+      }
```

Comment:
Couldn't we just query `if (!this.startingSyncTimestamp)`? Then we wouldn't need `this.firstTime`.